### PR TITLE
jtbl: add page

### PR DESCRIPTION
**Pull request by kellyjonbrazil**
Add new page for jtbl command

- [X] The page (if new), does not already exist in the repo.
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/jtbl.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+> Utility to print JSON and JSON Lines data as a table in the terminal.
+> More information: <https://github.com/kellyjonbrazil/jtbl>.
+
+- Print a table from JSON input:
+
+`cat {{file.json}} | jtbl`
+
+- Specify the column width for wrapping:
+
+`cat {{file.json}} | jtbl --cols={{number}}`
```
Comment (marchersimon) [inline pages/common/jtbl.md:12]:
```suggestion
- Specify the column width for wrapping:

`cat {{file.json}} | jtbl --cols={{width}}`
```

File: pages/common/jtbl.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+- Truncate rows instead of wrapping:
+
+`cat {{file.json}} | jtbl -t`
+
+- No wrapping or truncation of rows:
```
Comment (marchersimon) [inline pages/common/jtbl.md:18]:
```suggestion
- Don't wrap or truncate rows:
```

**Review by marchersimon — CHANGES_REQUESTED** [review]
Just a few suggestion. Also, since the description says you can also print JSON Lines data, does the tool autodetect the format or does it need some additional flag?

**Comment by kellyjonbrazil** [comment]
Thanks for the updates! `jtbl` autodetects JSON vs JSON Lines, so no other options are required.

**Comment by CleanMachine1** [comment]
Since this command works on windows, cat should probably be replaced with echo, since it makes it compatible. However using echo doesn't fit this page at all so I feel we need others opinions

Proof of windows support for anyone 
```
You can install jtbl via pip, via OS Package Repositories, MSI installer for Windows, or by downloading the correct binary for your architecture and running it anywhere on your filesystem.
```

**Review by marchersimon — APPROVED** [review]
Thanks @kellyjonbrazil

**Comment by marchersimon** [comment]
@CleanMachine1 I think replacing cat by any other command would probably confuse more than it would help anyone using windows. (Personally I think that we should focus on other platforms, since those are the ones tldr is used on)

**Comment by kellyjonbrazil** [comment]
I think you would use `type` in windows vs. `cat` in linux/etc., but it might get too wordy to include that.

File: pages/common/jtbl.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+`cat {{file.json}} | jtbl --cols={{width}}`
+
+- Truncate rows instead of wrapping:
+
+`cat {{file.json}} | jtbl -t`
+
+- Don't wrap or truncate rows:
+
+`cat {{file.json}} | jtbl -n`
```
Comment (CleanMachine1) [inline pages/common/jtbl.md:20]: I like when descriptions tell you exactly what they do, rather than just making the user infer.

So for example command 2 `Print a table from JSON or JSON Lines input and truncate rows instead of wrapping them:

Comment (CleanMachine1) [reply pages/common/jtbl.md:20]: Sure.

**Review by CleanMachine1 — APPROVED** [review]

**Review by nicokosi — APPROVED** [review]
👍
PS: I did not know the "JSON Lines" term (but I knew the concept). 🤓

**Comment by nicokosi** [comment]
> I think you would use `type` in windows vs. `cat` in linux/etc., but it might get too wordy to include that.

Good point. Not sure it's too wordy. What about something like:

> Utility to print JSON and JSON Lines data as a table in the terminal, reading from the standard input (for instance, via `cat` on Linux/macOS or `type` on Windows).
 > More information: <https://github.com/kellyjonbrazil/jtbl>.

**Comment by mfrw** [comment]
> > I think you would use `type` in windows vs. `cat` in linux/etc., but it might get too wordy to include that.
> 
> Good point. Not sure it's too wordy. What about something like:
> 
> > Utility to print JSON and JSON Lines data as a table in the terminal, reading from the standard input (for instance, via `cat` on Linux/macOS or `type` on Windows).
> > More information: https://github.com/kellyjonbrazil/jtbl.

This seems a little wordy to me ... lets see what other folks have to say

**Comment by CleanMachine1** [comment]
do these flags have long versions --

**Comment by nicokosi** [comment]
> do these flags have long versions --

No.

**Comment by CleanMachine1** [comment]
oh cool, you made this! i didn't see that, sorry.

**Review by sbrl — APPROVED** [review]
Thanks, @kellyjonbrazil!
